content/javascript/concepts/npm/npm.md

```diff
@@ -15,41 +15,63 @@ CatalogContent:
 
 ## Packages
 
-Packages are tools composed of files and folders of JavaScript code. They provide one or more operations that can be installed, imported and used in a separate project. Thanks to NPM, the installation process for packages is uniform.
+Packages are tools composed of files and folders of JavaScript code. They provide one or more operations that can be installed, imported, and used in a separate project. Thanks to npm, the installation process for packages is uniform.
 
-## The NPM Registry
+## The npm Registry
 
-NPM comes as two primary tools:
+npm comes as two primary tools:
 
 - A command-line tool with abilities that include installing and uninstalling packages.
-- An online registry of all packages registered with NPM.
+- An online registry of all packages registered with npm.
 
-On [the NPM registry](https://www.npmjs.com/), all public packages can be searched and reviewed before installing. Most packages come with a **README.md** file with instructions for installation and usage.
+On [the npm registry](https://www.npmjs.com/), all public packages can be searched and reviewed before installation. Most packages come with a `README.md` file with instructions for installation and usage.
 
 ## Common Commands
 
-As a command-line tool, NPM comes with a litany of commands for working with packages. One of the most common NPM commands is:
+As a command-line tool, npm comes with a litany of commands for working with packages. One of the most common npm commands is:
 
-```bash
+```shell
 npm install <package-name-here>
 ```
 
-This is used to install packages from the NPM registry and add them to a JavaScript project. Running the command as `npm install` with no specific package will (re)install all the packages for the project.
+This is used to install packages from the npm registry and add them to a JavaScript project. Running the command as `npm install` with no specific package will (re)install all the packages for the project.
 
-To learn more, run the following to get a brief overview of common commands:
+To learn more, run the following command to get a brief overview of common commands:
 
-```bash
+```shell
 npm --help
 ```
 
-`npm -h` works, too.
+> **Note:** `npm -h` works, too.
 
-Alternatively, run the following to display a (long) list of all NPM commands paired with brief descriptions:
+Alternatively, a (long) list of all npm commands, paired with brief descriptions, can be viewed with the `-l` flag:
 
-```bash
+```shell
 npm -l
 ```
+
+## Global Packages
+
+npm also features the ability to install packages globally, so packages can be used as command-line tools anywhere.
+
+The `-g` flag installs a package globally:
+
+```shell
+npm install -g <package-name-here>
+```
+
+Additionally, the `npx` executes packages anywhere while also handling installation. Usage:
```

Comment:
This doesn't seem right? Isn't `npx` meant to execute packages without having to install (as opposed to handling it)?